### ISSUE TITLE
Share Button not working in IE11

### ISSUE THREAD

**Issue by pradheepap** [carrot/share-button#315]
## Details
   The network buttons that appear after clicking share button does not disappear even after clicking the facebook/twitter buttons in IE11. It works fine in chrome and FF
#### OS(s)

- ​

#### Browser(s)

- ​IE 11

### Steps to Reproduce

[jsfiddle Example](http://jsfiddle.net/28fn23bj/)

1. ​Click Share button in the demo page (index.html)
2. ​ The network buttons appear.
3. ​Click on any of the network except mailto.
4. The network page (facebook/twitter/linkedin) opens. But the image doesn't disappear

### TL; DR

...

## Comments

Anything else related to the issue that doesn't fit above.
